flox-bash/Makefile

```diff
@@ -107,7 +107,7 @@ SRC = $(SRC_BASH) $(SRC_BATS) $(SRC_JQ)
 SHFMT = shfmt --language-dialect bash
 # SC1090 (warning): ShellCheck can't follow non-constant source. Use a directive to specify location.
 # SC1091 (info): Not following: ./init.sh was not specified as input (see shellcheck -x).
-# SC2034 (warning): floxMetricsConsent appears unused. Verify use (or export if used externally).
+# SC2034 (warning): floxFooBar appears unused. Verify use (or export if used externally).
```

Comment:
??

Reply:
Yeah that comment was just a sample warning returned by shellcheck when it was initially run, and now that we're removing that variable I similarly removed mention of it from the comment. Probably best to just put it back so it doesn't appear in the diff.